lib/kubernetes-deploy/logger.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+require 'logger'
+
+module KubernetesDeploy
+  module Logger
+    def self.included(base)
```

Comment:
We already have ActiveSupport as a dependency, in case you want to use `AS::Concern` (I don't have any strong opinion about using it)